component/background/StorageService.tsx

```diff
@@ -28,9 +28,9 @@ class StorageService {
             data[STORAGE_KEY_ID] = this.generateUUID();
             this.storage.set(data, () => {
               this.status = true;
-              resolve();
             });
           }
+          resolve();
```

Comment:
Bug Fix: If `id` is set in storage, then we would never `resolve` (no then can be fired). We need to move out of the `if` block.